cmplibrary/src/test/java/com/sourcepoint/gdpr_cmplibrary/ConsentLibBuilderTest.java

```diff
@@ -83,7 +85,7 @@ private Field getDeclaredFieldAccess(String fieldName) throws Exception {
 
     @Test
     public void ConsentLibBuilder(){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.